modules/persistent-storage-csi-efs-metrics-procedure-cli.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-procedure-cli_{context}"]
+= Enabling usage metrics using the CLI
+
+To enable AWS EFS storage CSI usage metrics using the CLI:
+
+. Edit the `ClusterCSIDriver` YAML file by running the following command:
++
+[source, terminal]
+----
+$ oc edit clustercsidriver efs.csi.aws.com -o yaml
+----
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value from `Disabled` (default) to `RecursiveWalk`.
++
+`RecursiveWalk` indicates that volume metrics collection in the AWS EFS CSI Driver is performed by recursively walking through the files in the volume.
+
+. (Optional) To define how the recursive walk operates, you can also set the following fields:
```

Comment:
Per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks):

```suggestion
. Optional: To define how the recursive walk operates, you can also set the following fields:
```